src/ops.c

```diff
@@ -249,18 +256,34 @@ shift_line(
 	}
 	else
 	    i += amount;
-	count = (long long)i * (long long)sw_val;
+	count = i * sw_val;
```

Comment:
`(long long)` cast removed.

Reply:
What is the `sizeof(long)` on VAX32?

Reply:
On VAX I got the following:
The size of int  is: 4 bytes
The size of long is: 4 bytes
long long does not exist.